### ISSUE TITLE
[Bug] Random crashes

### ISSUE THREAD

**Issue by timothykumm** [zyrouge/symphony#21]
Music was playing in schuffle, crashed after song finished and next song was going to start
![Screenshot_20230116-172426](https://user-images.githubusercontent.com/27158937/212726345-a1fc2a8e-0820-42ef-bafc-cd0e23370c37.png)

**Comment by zyrouge** [comment]
> Music was playing in schuffle, crashed after song finished and next song was going to start ![Screenshot_20230116-172426](https://user-images.githubusercontent.com/27158937/212726345-a1fc2a8e-0820-42ef-bafc-cd0e23370c37.png)

Are you using the latest version?

**Comment by timothykumm** [comment]
Yes I am using the latest version.

**Comment by zyrouge** [comment]
Does this occur frequently? If so, could you provide me steps to reproduce this issue? I'll try to figure out what's wrong but those steps would be helpful.

**Comment by timothykumm** [comment]
> Does this occur frequently? If so, could you provide me steps to reproduce this issue? I'll try to figure out what's wrong but those steps would be helpful.

I realised that it does occur every few days but most of the times the error warning does not appear and it just gives me a white screen. I managed to reproduce the bug. The application crashes if I doubleclick or trippleclick a song... So multiple clicks on a song makes the music player crash on my phone (Google Pixel 7 Pro, running Android 13).

Strangely when I initially reported this bug, the crash occured while my phone was in my pocket and I was listening to music in the background. But now I am not really sure about it.
